src/commands/lint.ts

```diff
@@ -86,13 +86,31 @@ export default class Lint extends Command {
   async run(): Promise<void> {
     const { argv, flags } = this.parse(Lint);
 
-    if (flags.output !== '-' && flags.output !== '-2') {
-      //Warn user
-      this.log(
-        yellow(
-          'You are using a hidden command. This command is not intended for public consumption yet. It might be broken, hard to use or simply redundant. Tread with care.'
-        )
+    let files: string[] = [];
+    if (!argv || argv.length === 0) {
+      const superPath = await detectSuperJson(process.cwd());
```

Comment:
Wondering whether author would use this command in more nested folders, don't we want to add `--scan` flag as `install` command has?

Reply:
Yeah, that's good point.